locales/es_ES/pcsx2_Main.po

```diff
@@ -1239,14 +1236,12 @@ msgid "Welcome to PCSX2!"
 msgstr "¡Bienvenido a PCSX2!"
 
 #: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:106
-#, fuzzy
 msgid "Configuration Guide"
-msgstr "Guías de configuración (internet)"
+msgstr "Guía de configuración"
 
 #: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:110
-#, fuzzy
 msgid "Readme / FAQ"
-msgstr "Léeme / Preguntas frecuentes (desconectado/PDF)"
+msgstr "Léeme/Preguntas frecuentes"
```

Comment:
Are you sure the empty space must be gone?

Reply:
In this case, the slash is used to separate two possible options or choices, so RAE says spaces shouldn't be applied here. Would be point 1. b) in here: https://www.rae.es/dpd/barra

Reply:
When the slash is used to offer options, according to RAE, no spaces should be in between the slash. It should be rule 1. b) here: https://www.rae.es/dpd/barra

And you can also see examples in Wikilengua: https://es.wikipedia.org/wiki/Barra_(tipograf%C3%ADa)